plugins/inputs/prometheus/README.md

```diff
@@ -33,6 +33,10 @@ in Prometheus format.
   ## - prometheus.io/path: If the metrics path is not /metrics, define it with this annotation.
   ## - prometheus.io/port: If port is not 9102 use this annotation
   # monitor_kubernetes_pods = true
+  ## Get the list of pods to scrape from either:
+  ##    - version 1 (default): the kubernetes watch api (cluster-wide)
+  ##    - version 2: the local cadvisor api (node-wide); for scalability. Note that the environment variable NODE_IP must be set to the host IP.
+  # monitor_kubernetes_pods_version = 1
```

Comment:
What do you think about giving this a more meaningful name like:
`pod_scrape_scope = "cluster"` and `pod_scrape_scope = "node"`